migrations/v2.js

```diff
@@ -0,0 +1,125 @@
+import { describe, getCourse, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Glossary - v2.0.3 to v2.1.0', async () => {
+  // https://github.com/adaptlearning/adapt-contrib-glossary/compare/v2.0.3..v2.1.0
+
+  let course;
+
+  whereFromPlugin('Glossary - from v2.0.3', { name: 'adapt-contrib-glossary', version: '>=2.0.0 <2.1.0' });
```

Comment:
```suggestion
  whereFromPlugin('Glossary - from v2.0.3', { name: 'adapt-contrib-glossary', version: '<2.1.0' });
```

Reply:
v2.0.0 is the earliest version to support fw v2, shouldn't were therefore constrain to >=2.0.0 @joe-replin ?